### PR TITLE
Fix cmake install

### PR DESCRIPTION
**Pull request by markus-oberhumer**

### CODE REVIEW

**Comment by markus-oberhumer** [comment]
I've just pushed a new version - this should also include the functionality of #222 .

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 disallow_intree_builds()
 
-project (utf8proc C)
+if (POLICY CMP0048)
+  cmake_policy (SET CMP0048 NEW)
+endif ()
+project (utf8proc VERSION 2.6.1 LANGUAGES C)
 
 # This is the ABI version number, which may differ from the
 # API version number (defined in utf8proc.h).
```
Comment (stevengj) [inline CMakeLists.txt:13]: Maybe add a comment here to remind us that the API version number should also be set in the `project` statement above when it is updated in utf8proc.h.

Comment (markus-oberhumer) [reply CMakeLists.txt:13]: > Maybe add a comment here to remind us that the API version number should also be set in the `project` statement above when it is updated in utf8proc.h.

Might be a good idea - please merge this and then adapt to your liking.

**Comment by markus-oberhumer** [comment]
Please merge this - the current git is broken as it installs headers into a directory called `include64`.

Furthermore this would also fix the following issues: #198, #200, #221, #222

**Pull request merged**